### PR TITLE
documentation: update predict_fn implementation for PyTorch EIA 1.5.1

### PR DESCRIPTION
**Pull request by tracycxw**
*Issue #, if available:*

*Description of changes:*
documentation: update predict_fn and model_fn implementation for PyTorch EIA 1.5.1

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ x ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ x ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: doc/frameworks/pytorch/using_pytorch.rst [hunk 1]
```diff
@@ -547,6 +548,26 @@ block, for example:
         with torch.jit.optimized_execution(True, {"target_device": "eia:0"}):
             output = model(input_data)
 
+If you use Elastic Inference with torch version >= 1.5.1, please implement your own ``predict_fn`` like below.
+
+.. code:: python
+
+    import torch
+    import numpy as np
+
```
Comment (metrizable) [inline doc/frameworks/pytorch/using_pytorch.rst:557]: nit [pep8](https://www.python.org/dev/peps/pep-0008/#blank-lines): two blank lines surrounding top-level functions (i know it's in the docs, but we should promote best practices there, too)

File: doc/frameworks/pytorch/using_pytorch.rst [hunk 2]
```diff
@@ -547,6 +548,26 @@ block, for example:
         with torch.jit.optimized_execution(True, {"target_device": "eia:0"}):
             output = model(input_data)
 
+If you use Elastic Inference with torch version >= 1.5.1, please implement your own ``predict_fn`` like below.
+
+.. code:: python
+
+    import torch
+    import numpy as np
```
Comment (metrizable) [inline doc/frameworks/pytorch/using_pytorch.rst:556]: nit: apply [`isort`](https://pypi.org/project/isort/) to imports

**Review by metrizable — CHANGES_REQUESTED** [review]

File: doc/frameworks/pytorch/using_pytorch.rst (hunk 1 above)
Comment (tracycxw) [reply doc/frameworks/pytorch/using_pytorch.rst:558]: fixed in rev2

File: doc/frameworks/pytorch/using_pytorch.rst (hunk 2 above)
Comment (tracycxw) [reply doc/frameworks/pytorch/using_pytorch.rst]: fix in rev2

**Comment by ajaykarpur** [comment]
The documentation linter is failing on this line:
```
/codebuild/output/src201229490/src/github.com/aws/sagemaker-python-sdk/doc/frameworks/pytorch/using_pytorch.rst:557: D002 Trailing whitespace
```

**Review by ajaykarpur — APPROVED** [review]

**Comment by tracycxw** [comment]
please hold the PR for now.. I have one more thing to edit

**Review by ajaykarpur — APPROVED** [review]

**Review by metrizable — APPROVED** [review]

**Pull request merged**